priority_variables_transform/ARIC/cause_of_death.yaml

```diff
@@ -4,11 +4,79 @@
       slot_derivations:
         associated_participant:
           populated_from: phv00203839
-        associated_visit:
-          value: ARIC Death Event
-          range: string
+        age_at_death:
+          expr: {phv00203832} * 365
         cause_of_death:
-          populated_from: phv00203820          
+          object_derivations:
+          - class_derivations:
+              CauseOfDeath:
+                populated_from: pht004049
+                slot_derivations:
+                  cause: phv00203820
+                  order: 1
```

Comment:
This section isn't correct:
```
                slot_derivations:
                  cause: phv00203820
                  order: 1
```
Needs to have the slot values specified in sub-keys like this:
```
                slot_derivations:
                  cause: 
                    populated_from: phv00203820
                  order:
                    value: 1
```
This needs to be fixed on all of the below as well.